templates/blog/post_detail.html

```diff
@@ -0,0 +1,48 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+<div class="container my-4">
+  <div class="post-detail">
+    <h2 class="mb-3">{{ post.title }}</h2>
+    <p class="text-muted">Автор: {{ user.username }} - {{ post.created_time|date:"d M Y" }}</p>
+    <p>{{ post.content }}</p>
+
+    <hr>
+
+    <h3 class="mt-4">Коментарі -  {{ post.commentary.all|length }}</h3>
+    {% if comments %}
+        <ul class="list-group">
+            {% for comment in comments %}
+                <li class="list-group-item">
+                  <strong>{{ comment.user.username }}</strong> прокоментував:
+                  <p>{{ comment.content }}</p>
+                </li>
+            {% endfor %}
+        </ul>
+    {% else %}
+        <p class="text-muted">Ще не залишено жодного коментаря.</p>
+    {% endif %}
+
+    <hr>
+
+    {% if user.is_authenticated %}
+        <form method="post" action="{% url "blog:comment-create" post.pk %}" class="mt-3">
+          {% csrf_token %}
+          <div class="mb-3">
+            <label>
+              <textarea name="content"
+                        class="form-control"
+                        rows="3"
+                        placeholder="Додати коментар..."
+                        required></textarea>
+            </label>
+          </div>
+          <button type="submit" class="btn btn-primary">Опублікувати коментар</button>
+        </form>
```

Comment:
Consider using crispy forms to enhance the appearance of the comment form. You can do this by using the `crispy` filter in the form rendering, for example: `{{ form|crispy }}`. Ensure the form is passed correctly from the view to the template.